common/test/java/com/couchbase/lite/CollectionTest.kt

```diff
@@ -113,16 +120,28 @@ class CollectionTest : BaseCollectionTest() {
         assertEquals(0, testCollection.count)
     }
 
+    // Test getting doc count from a collection in a deleted database returns 0
+    @Test
+    fun testGetDocCountFromCollectionInDeletedDatabase() {
+        val docID = "doc_id"
```

Comment:
Do you want to use saveSingleDocInCollectionWithID (or, the renamed version) here?